scripts/build/build/targets.py

```diff
@@ -257,6 +258,7 @@ def BuildEfr32Target():
     target.AppendModifier('rs911x', enable_rs911x=True).OnlyIfRe('-wifi')
     target.AppendModifier('wf200', enable_wf200=True).OnlyIfRe('-wifi')
     target.AppendModifier('wifi_ipv4', enable_wifi_ipv4=True).OnlyIfRe('-wifi')
+    target.AppendModifier('917-soc', enable_917_soc=True).OnlyIfRe('-wifi')
```

Comment:
This doesn't fit with the standard use for the modifiers. Either use `917_soc` or `917`.

Reply:
renamed to `917_soc` but this is a workaround the main fix might take longer to support and testing.